packages/form-js-viewer/src/render/components/Util.js

```diff
@@ -2,7 +2,7 @@ import snarkdown from 'snarkdown';
 
 import { sanitizeHTML } from './Sanitizer';
 
-export function formFieldClasses(type, errors = []) {
+export function formFieldClasses(type, disabled = false, errors = []) {
```

Comment:
Why do we need the extra class if we go for the simple solution?

Reply:
I removed the commit. 👍🏻